### ISSUE TITLE
Resistances are shown as more than max value

### ISSUE THREAD

**Issue by xafgun** [suriyun-production/mmorpg-kit-docs#1120]
Hello,

Resistances are shown as more than max value if you set resistance higher than max resistance.

For fix that.

GameDataHelper.cs - 121 and 123. line like be this.

        /// <summary>
        /// Combine resistance amounts dictionary
        /// </summary>
        /// <param name="targetDictionary"></param>
        /// <param name="newEntry"></param>
        /// <returns></returns>
        public static Dictionary<DamageElement, float> CombineResistances(Dictionary<DamageElement, float> targetDictionary, KeyValuePair<DamageElement, float> newEntry)
        {
            if (targetDictionary == null)
                targetDictionary = new Dictionary<DamageElement, float>();
            if (newEntry.Key != null)
            {
                if (!targetDictionary.ContainsKey(newEntry.Key))
                    targetDictionary[newEntry.Key] = System.Math.Min(newEntry.Value, newEntry.Key.maxResistanceAmount);
                else
                    targetDictionary[newEntry.Key] += System.Math.Min(newEntry.Value, newEntry.Key.maxResistanceAmount);
            }
            return targetDictionary;
        }

**Comment by insthync** [comment]
Thank you, I will change it to
```
        public static Dictionary<DamageElement, float> CombineResistances(Dictionary<DamageElement, float> targetDictionary, KeyValuePair<DamageElement, float> newEntry)
        {
            if (targetDictionary == null)
                targetDictionary = new Dictionary<DamageElement, float>();
            if (newEntry.Key != null)
            {
                if (!targetDictionary.ContainsKey(newEntry.Key))
                    targetDictionary[newEntry.Key] = newEntry.Value;
                else
                    targetDictionary[newEntry.Key] += newEntry.Value;
                if (targetDictionary[newEntry.Key] > newEntry.Key.maxResistanceAmount)
                    targetDictionary[newEntry.Key] = newEntry.Key.maxResistanceAmount;
            }
            return targetDictionary;
        }
```

**Comment by insthync** [comment]
Fixed in 1.69
